docs/rootless.md

```diff
@@ -56,9 +56,10 @@ $ build-using-dockerfile --buildkit-addr unix:///run/user/1001/buildkit/buildkit
 
 ## Set up (using a container)
 
+Docker image is available as [`moby/buildkit:master-rootless`](https://hub.docker.com/r/moby/buildkit/tags/).
```

Comment:
Don't we want to promote `moby/buildkit:rootless` (and then describe master as alternative)